src/Service/Azure.DataApiBuilder.Service.csproj

```diff
@@ -45,12 +45,6 @@
     </ItemGroup>
 
 
-    <ItemGroup>
-        <!-- extends watching group to include .json and .gql files -->
-        <Watch Include="**\*.json;**\*.gql" Exclude="obj\**\*;bin\**\*" />
```

Comment:
woudln't we still want `Exclude="obj\**\*;bin\**\*"`

Reply:
those are automatically excluded, `Exclude` property itself requires `Include`, since we are not encluding anything, we don't have to exclude explicitly as well.